neuronsimulator/templates/neuronsimulator/plot_neural_responses.html

```diff
@@ -0,0 +1,140 @@
+{% load static %}
+{% block title %}Neuron Responses{% endblock %}
+
+{% block head_extras %}
+    {{ block.super }}
+    <link href="{% static 'vanillaSelectBox/vanillaSelectBox.css' %}" rel="stylesheet">
+
+    <style>
+        .select-zone {
+            display: block;
+            width: 100%;
+            min-height: 200px;
+        }
+        label {
+            display: inline-block;
+            min-width: 300px;
+            width: 300px;
+        }
+    </style>
+{% endblock %}
+
+{% block content %}
+    <div class="row">
+        <div class="col-6">
+            <h5>Select or enter input parameters for plotting neural responses:</h5>   
+            <br>
+            <div class="select-zone">
+                <label>Select a stimulated neuron:</label>
+                <select id="one-neuron">
+                    {% for neuron in neurons %}
+                        <option value="{{ neuron.name }}">{{ neuron.name }}</option>
+                    {% endfor %}
+                </select>
+            </div>
+
+            <div class="select-zone">
+                <label>Select one or more neuron(s) for calculating responses:</label>
+                <select id="multi-neurons" multiple="" size="12" style="display: inline-block;">
+                    {% for neuron in neurons %}
+                        <option value="{{ neuron.name }}">{{ neuron.name }}</option>
+                    {% endfor %}
+                </select>
+            </div>
+            <br>
+            <div>
+            {% if form %}
+                <form action="" method="POST" id="form" onsubmit="getSelectedValues()">
+                    {% csrf_token %}
+                    <!--handle errors-->
+                    {% if form.errors %}
+                        <!--non-field errors-->
+                        {% for error in form.non_field_errors %}
+                            <div class="alert alert-danger">{{ error }}</div>
+                        {% endfor %}
+                    {% endif %}
+
+                    <!--hidden fields-->
+                    {% for hidden_field in form.hidden_fields %}
+                        {% if hidden_field.errors %}
+                            <div class="alert alert-danger">{{ hidden_field.errors }}</div>
+                        {% endif %}
+                        {{ hidden_field }}
+                    {% endfor %}
+                    <!--non-hidden fields-->
+                    {% for field in form.visible_fields %}
+                        <div class="form-control">
+                            <label>{{ field.label }}</label>
+                            {% if field.errors %}
+                                <div class="alert alert-danger">{{ field.errors }}</div>
+                            {% endif %}
+                            {{ field }}
+                            {% if field.help_text %}
+                                <small class="form-text">{{ field.help_text }}</small>
```

Comment:
C. I often use `<small>` tags like this, and have gotten feedback that it should be done with CSS.